### DISCUSSION TITLE
Lazer Editor: Slider Velocity

### DISCUSSION THREAD

**Discussion by Banana090** [ppy/osu#31157] [Q&A]
I'm very new mapper, so maybe I'm missing on some of the hotkeys etc. but very often I do want to create slider with velocity different from what it was before while using drawing feature to draw slider
By default, when I try to place the new slider it automatically chooses the velocity of the previous slider on the timeline, which I do not want in that case. I want to be able to change the velocity of slider I'm about to draw, but it doesn't let me click on velocity setting on the timeline of non-yet-placed object.

So, I have two options here:
1. Place a slider -> Change it's velocity -> Create slider shape by hand
2. Place a new slider on a timeline -> Change it's velocity to what I want it to be -> move it somewhere to the left -> Create a new slider -> Use drawing feature with desired velocity

Proposals on fix:
1. Add a velocity slider setting in the inspector
![image](https://github.com/user-attachments/assets/5fb45845-77ad-4969-aa09-07975915629d)
2. Make velocity popup button clickable on the timeline for preview objects
![image](https://github.com/user-attachments/assets/1325f835-0c77-4668-893f-7fff7e9605f2)
3. Create a hotkey (like ctrl+scroll) for changing slider velocity

**Comment by peppy** [comment]
This was brought up in private feedback recently. I can agree with adding back a slider bar to the inspector area.
